README.md

```diff
@@ -1 +1,58 @@
-# To-do-list
\ No newline at end of file
+# To-do-list
+Creating the To do list app
+
+>In this project I´m creating the "to-do list"app where I had to add tasks, to be more specific I had to add the title of the task, so far this app has the function to add and remove tasks.
+## Built With
+
+- Html
+- Css
+- JavaScript
+- Webpack
+- Npm dependencies
+- Linters
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+Copy the SSH code to download all the files.
+
+### Prerequisites
+
+Github.
+Code editor.
+
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
+
```

Comment:
- [ ] Kindly add information in these :point_up:  sections in your README file so that it may look more professional. add instruction to clone, install dependencies(`npm install`), and run the project locally(`npm start`).